### ISSUE TITLE
Tests?

### ISSUE THREAD

**Issue by wied03** [webpack/karma-webpack#114]
Is there a plan to start having tests for this project?

**Comment by wied03** [comment]
Shameless self promotion, but I took some of the good ideas from webpack dev middleware and this tool and did a fresh start with TDD (at least at the integration level). Maybe it might be of use - https://www.npmjs.com/package/karma-webpack_2

**Comment by d3viant0ne** [comment]
@wied03 - Project validations will be coming in this week. Leaving a note here for @MikaAK @goldhand and @d3viant0ne to take a moment to have a look at your implementation. Just skimming it for a few minutes, that is closer to the direction I would have gone ( less the memoryfs ).

Mika has some ideas as do myself and Will, give us a week or so and we will get this whipped into shape.

I'm going to close this for now. All the validation work will be handled by the team in #140

**Comment by wied03** [comment]
@d3viant0ne - I certainly could consider folding my work in. I just wanted something that was test driven to depend on for my project.

**Comment by d3viant0ne** [comment]
@wied03 - It's been discussed. @MikaAK will be opening an issue to bring the maintainers of the two major `karma-webpack` alternatives to the table so we can try and get the best of everything into a single solution. We just need to work through the `tech debt` and get code validations in place before we start adding new features.

The above is planned for the next semver `Major` with all the details needing to be ironed out prior of course. A release will be cut this week, the build & test base has already landed, then the tests & any PRs to stabilize the current version followed by feature / fix planning for the next Major.

Short version: Assuming all parties are willing, we will most certainly be including both you & @aaronjensen in the changes to come.
